Modules/Core/Common/include/itkSpatialOrientation.h

```diff
@@ -81,8 +79,7 @@ typedef enum
 // co-registered and pieced together.... PD-T2 interleaving of
 // slices is handled with choosing
 // which spectrum to load via the prototypical file name.
-typedef enum
-{
+using ValidCoordinateOrientationFlags = enum {
```

Comment:
Reverting this and similar changes below in that file (Modules/Core/Common/include/itkSpatialOrientation.h) allows  build on _Linux_ , tried to build the branch on _Debian 10, GCC 8.3.1_, default configuration without tests. Otherwise build fails with:

```
[ 82%] Building CXX object Modules/IO/IPL/src/CMakeFiles/ITKIOIPL.dir/itkIPLCommonImageIO.cxx.o
In file included from /home/r/tmp/itk/ITK/Modules/IO/IPL/include/itkIPLCommonImageIO.h:38,
                 from /home/r/tmp/itk/ITK/Modules/IO/IPL/src/itkIPLCommonImageIO.cxx:20:
/home/r/tmp/itk/ITK/Modules/IO/IPL/include/itkGEImageHeader.h:41:8: warning: ‘GEImageHeader’ has a field ‘GEImageHeader::coordinateOrientation’ whose type has no linkage [-Wsubobject-linkage]
 struct GEImageHeader
        ^~~~~~~~~~~~~
In file included from /home/r/tmp/itk/ITK/Modules/IO/IPL/src/itkIPLCommonImageIO.cxx:22:
/home/r/tmp/itk/ITK/Modules/Core/Common/include/itkSpatialOrientationAdapter.h:81:24: warning: ‘itk::SpatialOrientationAdapter’ has a base ‘itk::OrientationAdapterBase<itk::SpatialOrientation::<unnamed enum>, 3>’ whose type uses the anonymous namespace [-Wsubobject-linkage]
 class ITKCommon_EXPORT SpatialOrientationAdapter
                        ^~~~~~~~~~~~~~~~~~~~~~~~~
/home/r/tmp/itk/ITK/Modules/Core/Common/include/itkSpatialOrientationAdapter.h:104:3: error: ‘virtual itk::OrientationAdapterBase<itk::SpatialOrientation::<unnamed enum>, 3>::DirectionType itk::SpatialOrientationAdapter::ToDirectionCosines(const OrientationType&)’, declared using unnamed type, is used but never defined [-fpermissive]
   ToDirectionCosines(const OrientationType & Or) override;
   ^~~~~~~~~~~~~~~~~~
/home/r/tmp/itk/ITK/Modules/Core/Common/include/itkSpatialOrientationAdapter.h:90:78: note: ‘using OrientationType = using ValidCoordinateOrientationFlags = enum itk::SpatialOrientation::<unnamed>’ does not refer to the unqualified type, so it is not used for linkage
   using OrientationType = SpatialOrientation::ValidCoordinateOrientationFlags;
                                                                              ^
/home/r/tmp/itk/ITK/Modules/Core/Common/include/itkSpatialOrientationAdapter.h:100:3: warning: ‘virtual itk::SpatialOrientationAdapter::OrientationType itk::SpatialOrientationAdapter::FromDirectionCosines(const DirectionType&)’ used but never defined
   FromDirectionCosines(const DirectionType & Dir) override;
   ^~~~~~~~~~~~~~~~~~~~
/home/r/tmp/itk/ITK/Modules/Core/Common/include/itkSpatialOrientationAdapter.h:104:3: warning: ‘virtual itk::OrientationAdapterBase<itk::SpatialOrientation::<unnamed enum>, 3>::DirectionType itk::SpatialOrientationAdapter::ToDirectionCosines(const OrientationType&)’ used but never defined
   ToDirectionCosines(const OrientationType & Or) override;
   ^~~~~~~~~~~~~~~~~~
make[2]: *** [Modules/IO/IPL/src/CMakeFiles/ITKIOIPL.dir/build.make:76: Modules/IO/IPL/src/CMakeFiles/ITKIOIPL.dir/itkIPLCommonImageIO.cxx.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:9362: Modules/IO/IPL/src/CMakeFiles/ITKIOIPL.dir/all] Error 2
make: *** [Makefile:156: all] Error 2
```

I don't want to step in into this PR, just FYI.